diff --git a/dist/hmr.cjs b/dist/hmr.cjs
index 9addf93f94594b3aac08f100528760c0a59e8a42..e39333f24a57b61fc9442a704ed99183ae098006 100644
--- a/dist/hmr.cjs
+++ b/dist/hmr.cjs
@@ -120,8 +120,6 @@ async function fetchUpdate(runner, { path, acceptedPath }) {
   return () => {
     for (const { deps, fn } of qualifiedCallbacks)
       fn(deps.map((dep) => dep === acceptedPath ? fetchedModule : void 0));
-    const loggedPath = isSelfUpdate ? path : `${acceptedPath} via ${path}`;
-    console.log(`${c.cyan("[vite-node]")} hot updated: ${loggedPath}`);
   };
 }
 function warnFailedFetch(err, path) {
diff --git a/dist/hmr.mjs b/dist/hmr.mjs
index d15345c847d91fd5018be6484bd660a42bc6b61a..bb07017d12958bf443332887af344a75d01a82c5 100644
--- a/dist/hmr.mjs
+++ b/dist/hmr.mjs
@@ -118,8 +118,6 @@ async function fetchUpdate(runner, { path, acceptedPath }) {
   return () => {
     for (const { deps, fn } of qualifiedCallbacks)
       fn(deps.map((dep) => dep === acceptedPath ? fetchedModule : void 0));
-    const loggedPath = isSelfUpdate ? path : `${acceptedPath} via ${path}`;
-    console.log(`${c.cyan("[vite-node]")} hot updated: ${loggedPath}`);
   };
 }
 function warnFailedFetch(err, path) {